phpBB/phpbb/console/command/reparser/reparse.php

```diff
@@ -163,29 +171,39 @@ protected function execute(InputInterface $input, OutputInterface $output)
 		$this->input = $input;
 		$this->output = $output;
 		$this->io = new SymfonyStyle($input, $output);
-		$this->load_resume_data();
 
-		$name = $input->getArgument('reparser-name');
-		if (isset($name))
+		if (!$this->reparse_lock->acquire())
```

Comment:
Is that conditional correct? Looks like it should not be negated.

Either way, I think it would be better to simply have a conditional with only one branch that throws an exception and no else, e.g..

```php
if (!$this->reparse_lock->acquire())
{
    throw new Exception;
}

// ...the rest executes as usual
```